content/about/cs312.rst

```diff
@@ -3,57 +3,53 @@ CS312 System Administration Course
 :slug: students/cs312
 :menu: Education, CS312, 4
 
-This course was created by Lance Albertson and Jeff Sheltren in 2009 who both
-worked at the `Oregon State University Open Source Lab`_. Lance is currently the
-Director, and Jeff was the Operations Manager at the time. Both Lance and Jeff
-have backgrounds in computer science and system administration.
+This course was originally created by Lance Albertson and Jeff Sheltren in 2009 who both worked at the `Oregon State
+University Open Source Lab`_. Lance is currently the Director, and Jeff was the Operations Manager at the time. Both
+Lance and Jeff have backgrounds in computer science and system administration. The course was revived in 2015 and 2016
+by Lance Albertson and Jordan Evans with more focus on newer technologies and methodologies.
 
 What is this Course?
 --------------------
 
-This is `CS312 - Linux System Administration`_, offered at `Oregon State
-University`_. The current plan is to offer this course once per year. Students
-in the course are expected to have at least a basic knowledge of UNIX/Linux
-operating systems. OSU students need to complete CS311 (Operating Systems)
-before they can enroll in this course. Exceptions can be made for students who
-can prove a strong knowledge of Linux.
+This is `CS312 - Linux System Administration`_, offered at `Oregon State University`_. The current plan is to offer
+this course once per year when OSL staff has time. Students in the course are expected to have at least a basic
+knowledge of UNIX/Linux operating systems. OSU students need to complete CS311 (Operating Systems) before they can
```

Comment:
@ramereth This might be helpful as a sub-section that says something like "Are there any prereqs for this course?"